lib/hanami/utils.rb

```diff
@@ -1,5 +1,8 @@
 # Hanami - The web, with simplicity
 #
+
+require 'pathname'
```

Comment:
sorry, I can't understand this line. Why we need to require `pathname` in base utils file? can we do it only in specifics files?

Reply:
got it, thanks for explanation!